src/clib/pio.h

```diff
@@ -595,7 +653,7 @@ extern "C" {
                             MPI_Comm io_comm, int *iosysidp);
     int PIOc_get_numiotasks(int iosysid, int *numiotasks);
     int PIOc_Init_Intracomm(MPI_Comm comp_comm, int num_iotasks, int stride, int base, int rearr,
-                            int *iosysidp);
+                            rearr_opt_t *rearr_opts, int *iosysidp);
```

Comment:
Yes I wonder if we should leave them out of the Init_Intracomm interface and have separate functions to set them - it seems like eventually we may want finer control of these variables and setting them in init functions limits that possibility.